stock_picking_move_line_no_print/readme/USAGE.md

```diff
@@ -0,0 +1,6 @@
+To hide operations from the delivery report:
+
+- Go to a delivery picking.
+- In the operations, click on the column selector to reveal the 🖶 column.
```

Comment:
Same problem as https://github.com/OCA/sale-workflow/pull/3733#discussion_r2113421656 and https://github.com/OCA/sale-workflow/pull/3733#discussion_r2113444892.